docs/rules/generator-star-spacing.md

```diff
@@ -75,6 +75,25 @@ An example of shorthand configuration:
 "generator-star-spacing": ["error", "after"]
 ```
 
+Additionally, this rule allows further configurability via overrides per function type.
+
+* `named` provides overrides for named functions
+* `anonymous` provides overrides for anonymous functions
+* `method` provides overrides for class methods or property function shorthand
+* `static` provides overrides for static class methods
```

Comment:
I included it simply because it is, technically, its own distinct case. I could imagine someone wishing to have methods as
```
class Foo {
    ...
    *method() {}
}
```
but static methods as
```
class Bar {
    ...
    static* method() {}
}
```

I don't necessarily feel strongly about it, however.